src/main/java/com/elasticpath/tools/mavenminimal/reactor/MavenDriver.java

```diff
@@ -145,6 +147,28 @@ private static CommandLine determineMavenCommand(final Reactor reactor, final Li
 		return cmdLine;
 	}
 
+
+	/**
+	 * The number of threads can be specified in several ways for Maven, each possibility needs to be removed.
+	 * This method
+	 * @param mavenArguments the arguments to pass to maven.
+	 */
+	static void removeThreadingFlags(final List<String> mavenArguments) {
+
+		mavenArguments.removeIf(s -> s != null && s.matches("(-T.+.*|--threads=.*)"));
```

Comment:
Would this change allow us to remove the iterator?

```suggestion
		mavenArguments.removeIf(s -> s != null && s.matches("(-T.+.*|--threads=.*|-T|--threads)"));
```

Reply:
No, because we need to remove two separate list items, not just one.

Reply:
Ohhhhh...ok, I get it.